scheduler/src/cook/kubernetes/api.clj

```diff
@@ -367,8 +367,14 @@
                   (log/info "In" compute-cluster-name "compute cluster, initializing pod watch helper")
                   (initialize-pod-watch-helper compute-cluster cook-pod-callback)
                   (catch Exception e
-                    (log/error e "Error during pod watch initial setup of looking at pods for" compute-cluster-name
-                               "and sleeping" reconnect-delay-ms "milliseconds before reconnect")
+                    (let [state @state-atom]
+                      (if (= state :deleted)
+                        (log/info
+                          e "In" compute-cluster-name "compute cluster, cannot create pod watch (cluster is deleted)"
+                          {:reconnect-delay-ms reconnect-delay-ms :state state})
+                        (log/error
+                          e "In" compute-cluster-name "compute cluster, error during pod watch initial setup"
```

Comment:
can we still leave the message saying that there's a sleep?

Reply:
👍🏼 Done